package.json

```diff
@@ -4,80 +4,83 @@
   "description": "Web UI for deploying code to kubernetes",
   "scripts": {
     "prebuild": "npm run build:clean",
-    "build:clean": "rimraf ./public/build/",
-    "build": "webpack --config ./public/webpack.config.js",
+    "build:clean": "./node_modules/.bin/rimraf ./public/build/",
+    "build": "./node_modules/.bin/webpack --config ./public/webpack.config.js",
     "watch": "npm run build -- --progress --colors --watch",
     "lint": "node_modules/.bin/eslint \"./public/src/**/*.js\"",
-    "lint:fix": "node_modules/.bin/eslint \"./public/src/**/*.js\" --fix | snazzy"
+    "lint:fix":
+      "node_modules/.bin/eslint \"./public/src/**/*.js\" --fix | snazzy"
   },
   "repository": {
     "type": "git",
     "url": "ssh://git@github.com/airware/vili.git"
   },
   "dependencies": {
     "browser-request": "0.3.3",
-    "firebase": "2.3.1",
     "human-size": "1.1.0",
-    "moment": "2.17.1",
-    "moment-duration-format": "1.3.0",
-    "prop-types": "15.5.10",
-    "react": "15.6.0",
-    "react-bootstrap": "0.31.0",
-    "react-bootstrap-typeahead": "0.8.4",
-    "react-dom": "15.5.4",
-    "react-redux": "4.4.5",
-    "react-router": "3.0.5",
-    "react-router-redux": "4.0.8",
-    "redux": "3.3.1",
-    "redux-thunk": "2.1.0",
-    "redux-ui": "0.0.15",
+    "immutable": "3.8.1",
+    "moment": "2.22.1",
```

Comment:
👍 This should address the GitHub Security Warnings we've been receiving.
CC @jasonmichaelhayes

Reply:
Yep, exactly